queries/highlights.scm

```diff
@@ -20,18 +23,29 @@
 "ternary" @keyword
 "optional" @keyword
 "actions" @keyword
+"default" @keyword
 "default_action" @keyword
 "return" @keyword
 "NoAction" @keyword
 "const" @keyword
 "meters" @keyword
 "counters" @keyword
-(direction) @keyword
-"bool" @type
-(bit_type) @type
+"_" @keyword
+"true" @constant
+"false" @constant
+(direction) @constant
```

Comment:
Why change directions from keywords to constants?

Reply:
I had seen this in another tree-sitter grammar around highlighting, but easy to fix back.